### ISSUE TITLE
Don't display vulgar and obscene sentences prominently.

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#2908]
I wonder if it might not be time to deal with the problem of vulgar or obscene sentences getting displayed.

### Several problems are ...

* Teachers may avoid using tatoeba.org with their students for this reason, not because the teacher is a prude, but because such material is not generally considered acceptable for school wokrk

* Some people may likely avoid the website for this reason, especially if this is what they see on their first visit.

* It likely makes tatoeba.org seem a little less friendly.

* It perhaps makes it seem like we don't care.  (Perhaps we shouldn't. I don't know.)

### Related Issue

https://github.com/Tatoeba/tatoeba2/issues/1328

### Recent Screenshot

<img width="707" alt="Screen Shot 2022-02-24 at 8 19 05" src="https://user-images.githubusercontent.com/7934273/155426808-78e3c9f4-96fa-4ac7-ba4b-df104349b85b.png">

**Comment by RobinvanderVliet** [comment]
I don't think we should hide such sentences outright, because that would make it easier for vandalism to go undetected. But maybe there could be an option in the settings to hide sentences with vulgarities? But does that mean we need store a list of vulgar expressions for each language here in the code?

**Comment by ckjpn** [comment]
I was wondering if having the incoming sentences as the default on the main page might not be such a good idea.
Having several buttons (or links) showing options to members and non-members might likely not only not so prominently display new sentences, which are more likely to be "trolling" types of sentences, those with political motivations, or added for shock value (vulgar or pornographic sentences.)

More of a "dashboard" type of home page, with several options to choose from, one of which could be to show incoming sentences, might be not only more useful for regular members and visitors, but could also eliminate this problem.

Here are several examples of ideas that I think could be adapted for use on tatoeba.org for this purpose.

1. My current dashboard for members.

Set up for TRANG, assuming she wanted to translate from Japanese to French
http://study.aitech.ac.jp/tatoeba/translate/links.php?f=jpn&t=fra&u=TRANG
Or, start here to set it up for yourself.
http://study.aitech.ac.jp/tatoeba/translate/links.php

2. An newer experiment:  Set up for English - Japanese - English

http://tatoeba.ueuo.com/dash/2.html

I thought that doing something like this might be good, with likely too many buttons at first. Then give members the choice in settings to choose which buttons to hide and show, might be useful, and also a way that one could more likely not display vulgar and other "trolling-like" sentences so prominently.  We could even choose a default subset of the "too many buttons" and in the settings allow members to remove or add buttons.

3. The following page, which was designed for translating English with audio, wouldn't be perfect for this, but the top of the page with a cut-down "advanced search" and several ways to find sentences to translate might make a good dashboard.

Set up for English with Audio to French
http://a4esl.org/temporary/tatoeba/translate/searches.php?t=fra
Or, start here to set it up for another language.
http://a4esl.org/temporary/tatoeba/translate/searches.php


Perhaps, maybe I should start another issue, suggesting creating a different main page for tatoeba.org, but for now, these are some of my thoughts on how to make tatoeba.org more user-friendly for school-age children and for various people around the world who find these kinds of sentences offensive.
